modules/programs/vim.nix

```diff
@@ -60,6 +61,11 @@ let
         optional (isString p && hasAttr p vpkgs) vpkgs.${p};
   in concatMap getPkg cfg.plugins;
 
+  optionalPlugins = let
+    vpkgs = pkgs.vimPlugins;
+    getPkg = p: [ p ];
+  in concatMap getPkg cfg.optionalPlugins;
+
```

Comment:
Isn't `optionalPlugins == cfg.optionalPlugins` at this point? I think we can just use `cfg.optionalPlugins` in the code below.